### DISCUSSION TITLE
Day 2 Retrospective - Please answer the following questions.

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#10] [Q&A]
What do we do well?

What could we have done better?

Ideas and suggestions?

**Comment by wicksipedia** [comment]
> What do we do well?

- Answering the questions in the chat as it went along
- Turning demo fails into learning opportunities
- Deeper dive into the reusable workflows (from the slack chat)
- Session length - 3hrs is about the limit for high level overview content

> What could we have done better?

- nothing

> Ideas and suggestions?

- links to labs/docs if we want to learn more in our own time
